src/foam/nanos/ticket/TicketNotificationNotificationCitationView.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.ticket',
+  name: 'TicketNotificationNotificationCitationView',
+  extends: 'foam.nanos.notification.NotificationCitationView',
+
+  imports: [
+    'stack',
+    'ticketDAO'
+  ],
+
+  requires: [
+    'foam.comics.v2.DAOControllerConfig',
+    'foam.u2.stack.StackBlock'
+  ],
+
+  css: `
+    ^link {
+      margin-top: 8px;
+      margin-left: 32px;
+  `,
+
+  properties: [
+    {
+      class: 'FObjectProperty',
+      of: 'foam.nanos.ticket.Ticket',
+      name: 'ticket',
+      hidden: true,
+      transient: true
+    },
+    {
+      class: 'Boolean',
+      name: 'hasFoundTicket',
+      documentation: 'Check if ticket id attached to the notification (data.ticket) is valid',
+      transient: true,
+      hidden: true
+    },
+    {
+      name: 'data',
+      postSet: function() {
+        if ( ! this.data.ticket ) {
+          this.hasFoundTicket = false;
+          return;
+        }
+
+        this.ticketDAO.find(this.data.ticket).then(ticket => {
+          this.hasFoundTicket = !! ticket;
+          this.ticket = this.hasFoundTicket ? ticket : undefined;
+        });
+      }
+    }
+  ],
+
+  methods: [
+    function render() {
+      const self = this;
+      this.SUPER();
+      this
```

Comment:
Can you inline this so that the button is not on a new line? Also set buttonStyle to 'LINK'